pkg/simple/client/network/options.go

```diff
@@ -2,14 +2,22 @@ package network
 
 import "github.com/spf13/pflag"
 
+type NSNPOptions struct {
+	AllowedIngressNamespaces []string `json:"allowedIngressNamespaces,omitempty" yaml:"allowedIngressNamespaces,omitempty"`
```

Comment:
DefaultAllowedIngressNamespaces

Reply:
This is not the default value, it can be set by the user, only if the user does not configure it then namespace ”kubesphere-moniting-system" is allowed by default.

Reply:
@zryfish Please take a look.

Reply:
I think we'd better set default values through configuration file, not in the code, cause we may need to change monitoring namespace name one day

Reply:
> I think we'd better set default values through configuration file, not in the code, cause we may need to change monitoring namespace name one day

done